composer.json

```diff
@@ -12,14 +12,18 @@
             "homepage": "http://sergei.lissovski.org/"
         }
     ],
+    "conflict": {
+        "doctrine/persistence": ">=2.0"

```

Comment:
Can you please elaborate on this one, why it is necessary?

Reply:
The `Doctrine\Common\Persistence\ManagerRegistry` class is deprecated since doctrine/persistence v1.3 and was removed in 2.0.